raspberrypi2-cxx11.cmake

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015, Alexandre Pretyman
+# All rights reserved.
+
+if(DEFINED POLLY_RASPBERRYPI2_CXX11_CMAKE)
+  return()
+else()
+  set(POLLY_RASPBERRYPI2_CXX11_CMAKE 1)
+endif()
+
+#Maybe this line should be in polly_init and not in polly_common?
```

Comment:
For now `polly_init` set variables like `POLLY_TOOLCHAIN_TAG` and `POLLY_TOOLCHAIN_NAME` and `polly_common`  check that this variables are set but I guess it can be improved in future. I'm planning to rethink/refactor a lot of stuff (like https://github.com/ruslo/polly/issues/31) and will remove this one too.